lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -121,13 +123,30 @@ def #{configuration[:column]}=(position)
 
             after_commit 'remove_instance_variable(:@scope_changed) if defined?(@scope_changed)'
 
-            scope :in_list, lambda { where(%q{#{quoted_table_name}.#{connection.quote_column_name(configuration[:column])} IS NOT NULL}) }
+            scope :in_list, lambda { where(%q{#{quoted_table_name}.#{quoted_position_column} IS NOT NULL}) }
+
+            def self.decrement_all_with_touch
+              update_all_with_touch %q(#{quoted_position_column} = (#{quoted_position_column} - 1))
+            end
+
+            def self.increment_all_with_touch
+              update_all_with_touch %q(#{quoted_position_column} = (#{quoted_position_column} + 1))
+            end
           EOV
 
           if configuration[:add_new_at].present?
             self.send :before_create, :"add_to_list_#{configuration[:add_new_at]}"
           end
+        end
 
+        private
+
+        def update_all_with_touch(updates)
+          attrs = new.send(:timestamp_attributes_for_update_in_model)
+          query = attrs.map { |attr| "#{attr} = :now" }
+          query.push updates
+          query = query.join(", ")
+          update_all([query, now: Time.now.utc])
```

Comment:
In Rails the method used to get the time is: current_time_from_proper_timezone. I think it might pay to act as close to the actual `touch` method as possible :)